pyproject.toml

```diff
@@ -9,6 +9,9 @@ select = [
 ]
 ignore = ["PLR0913"]
 
+[tool.ruff.pylint]
+max-branches = 13
```

Comment:
non-blocking: AWS SAM CLI has a max branch limit of 12, should we update this to be consistent?

Reply:
12 is the default but we had methods with 13. I didn't want to refactor within this PR so set it to 13 as the max instead.